etl/steps/data/meadow/irena/2024-11-01/renewable_capacity_statistics.py

```diff
@@ -0,0 +1,104 @@
+"""Load a snapshot and create a meadow dataset."""
+
+from typing import Dict
+
+import owid.catalog.processing as pr
+from owid.catalog import Table
+
+from etl.helpers import PathFinder, create_dataset
+
+# Get paths and naming conventions for current step.
+paths = PathFinder(__file__)
+# Expected sheet names.
+EXPECTED_SHEET_NAMES = ["Country", "All Data", "Regional", "Global", "About"]
+# Expected columns in the All Data sheet.
+EXPECTED_COLUMNS = [
+    "Region",
+    "Sub-region",
+    "Country",
+    "ISO3 code",
+    "M49 code",
+    "RE or Non-RE",
+    "Group Technology",
+    "Technology",
+    "Sub-Technology",
+    "Producer Type",
+    "Year",
+    "Electricity Generation (GWh)",
+    "Electricity Installed Capacity (MW)",
+    "Heat Generation (TJ)",
+    "Off-grid Biogas for Cooking (1,000 inhabitants)",
+    "Off-grid Biogas Production (1,000 m3)",
+    "Off-grid Electricity Access (1,000 inhabitants)",
+    "Public Flows (2021 USD M)",
+    "SDG 7a1 Intl. Public Flows (2021 USD M)",
+    "SDG 7b1 RE capacity per capita (W/inhabitant)",
+]
+
+
+def sanity_check_inputs(tables: Dict[str, Table]) -> None:
+    # Sanity checks.
+    error = "Sheet names have changed."
+    assert set(tables) == set(EXPECTED_SHEET_NAMES), error
+
+    error = "Columns have changed in the 'All Data' sheet."
+    assert set(EXPECTED_COLUMNS) == set(tables["All Data"].columns), error
```

Comment:
Great examples of how to use asserts better, really helpful for me to see, thanks!

Reply:
Thank you! :)